it/v1/endpoints/ListTransactionsControllerISpec.scala

```diff
@@ -24,8 +24,9 @@ import play.api.libs.ws.{WSRequest, WSResponse}
 import support.IntegrationBaseSpec
 import v1.models.errors._
 import v1.stubs.{AuditStub, AuthStub, DesStub, MtdIdLookupStub}
+import v1.fixtures.ListTransactionFixture._
 
-class RetrieveTransactionsControllerISpec extends IntegrationBaseSpec {
+class ListTransactionsControllerISpec extends IntegrationBaseSpec {
```

Comment:
For the DES JSON in this file the second transaction is missing a document ID (this will also require you to update the mtdJson val to include the transactionId)